src/main/sensors/gyro.c

```diff
@@ -577,6 +577,7 @@ void gyroUpdateSensor(gyroSensor_t *gyroSensor)
     for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
         // scale gyro output to degrees per second
         float gyroADCf = (float)gyroSensor->gyroDev.gyroADC[axis] * gyroSensor->gyroDev.scale;
+        DEBUG_SET(DEBUG_GYRO_UNFILTERED, axis, lrintf(gyroADCf));
```

Comment:
This seems to be capturing almost the same data point as `DEBUG_NOTCH` in https://github.com/betaflight/betaflight/pull/3965/files#diff-59167518290ca93ea0a3471df4e48bbbL594 , except for the dynamic filtering, which will probably be kept off while a user is tuning the static filters. Is this intentional, or should one of the two be removed?

Reply:
This is an extra debug mode to allow the capture of the unfiltered gyro data. The other debug modes are to help with filter tuning, this debug mode is to capture the raw gyro data after a crash. It's being added to help debug the "yaw spin to the moon" crash behaviour, but will also be useful in future for debugging any other funny behaviour that might occur after a crash.

Reply:
I think in the case of tuning the static filters with `DEBUG_NOTCH`, the values will be the same as `DEBUG_GYRO_UNFILTERED` in the case that dynamic filtering is turned off during tuning, so `DEBUG_NOTCH` should probably go, for the sake of less user confusion. (Even better, `DEBUG_GYRO_UNFILTERED` should be renamed to `DEBUG_NOTCH`, to avoid users having to learn something new.)

Reply:
`DEBUG_NOTCH` was always a misnomer. Currently I'm telling people who are helping debug crash behaviour to set the debug mode to `notch` which sounds silly. I'd like to change the name, but I'll leave that for the moment and go with your suggestions.

Reply:
Yes, it is a misnomer, as it describes what the collected data is used for (normally), not what it is. But since using `DEBUG_MODE = NOTCH` to tune their filters is probably the only time a lot of users use any kind of debug info (and the CLI), there are a lot of tuning guides out there that reference it, and renaming it will cause them to fail.
If we rename it early on in 3.3, the authors of tuning guides get a better lead in time before it gets changed in an official release.